.github/workflows/bundle-update.yml

```diff
@@ -0,0 +1,59 @@
+name: Scheduled Bundle Update PR
+
+on:
+  schedule:
+    - cron: '0 0 * * 2'
+  workflow_dispatch:
+
+jobs:
+  bundle-update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.4'
+
+      - name: Set up git
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
```

Comment:
I’d suggest setting up Git after “Run bundle update”, but it would mean more `if: env.has_update`s.